dartsim/src/ShapeFeatures.cc

```diff
@@ -328,6 +328,80 @@ AlignedBox3d ShapeFeatures::GetShapeAxisAlignedBoundingBox(
   return AlignedBox3d(box.getMin(), box.getMax());
 }
 
+/////////////////////////////////////////////////
```

Comment:
Maybe we should guard all these features with `DART_VERSION_AT_LEAST(6, 10, 0)`. What do you think @scpeters ?

Reply:
this is targeted at `ign-physics1`, which requires dart 6.10. I think we will need those guard conditions when we merge forward to `ign-physics2`.